odk1-src/collect-forms.rst

```diff
@@ -169,6 +169,9 @@ If you are connected to :doc:`an ODK Aggregate server  <collect-connect-aggregat
   - Use the `TRIM <https://support.google.com/docs/answer/3094140?hl=en>`_ function in the google sheets to remove the leading and trailing spaces from the cells.
   - Define empty cell in your tests to be a cell that is either empty or contains a single space.
 
+.. note::
+
+  Using Google Drive as a server, filled forms are sent to the first sheet in a given spreadsheet, no matter what is its name. If you use one spreadsheet to keep an original form and to collect filled forms make sure the sheet you expect to be filled is in the first place.
```

Comment:
"what is its name" -> "what its name is"

How about "an original form" -> "a form definition"? I had to think through what "original" would mean in this context.

Reply:
You are right, fixed. Thanks!